src/core/unit.h

```diff
@@ -45,7 +46,7 @@ typedef enum CollectMode {
 } CollectMode;
 
 static inline bool UNIT_IS_ACTIVE_OR_RELOADING(UnitActiveState t) {
-        return IN_SET(t, UNIT_ACTIVE, UNIT_RELOADING);
+        return IN_SET(t, UNIT_ACTIVE, UNIT_RELOADING, UNIT_REFRESHING);
```

Comment:
Please make sure this is treated as active everywhere, e.g. _IS_ACTIVE_OR_ACTIVATING below too